### ISSUE TITLE
Why there are noises in documents of 20000Hz wav audio?

### ISSUE THREAD

**Issue by RickPeng** [superpoweredSDK/Low-Latency-Android-Audio-iOS-Audio-Engine#133]
Why there are noises in documents of 20000Hz wav audio?

sampling rate:44,100
bit of each sample: 16
lasting time:00:30
audio channels:1
----------------------------------------
[audacity.wav.zip](https://github.com/superpoweredSDK/Low-Latency-Android-Audio-iOS-Audio-Engine/files/313128/audacity.wav.zip)

code 
samplerateString = "44100";
buffersizeString = "512";

**Comment by superpoweredSDK** [comment]
I don't understand the question, sorry. How do you play them? If they are at 20000 Hz (is that sample rate exist?), then do you resample for 44100 Hz? What is your output sample rate? Do you do fast mixer on Android? Please explain.

**Comment by RickPeng** [comment]
I am sorry for my bad English, it’s the first time I ask question from here.
 
Well, the problem is : 
I played a wav document on my android device with supoerPoweredSDK, 
here’s the data of the document:
Sample rate : 41100
Sample digit：16
Sound frequency: 20k HZ
 
But when I played it on android device I heard the sound of other frequencies,
which is not supposed to be there because we shouldn’t be able to hear this sound. 
So I think there’s something wrong with it.
Thanks for replying and taking up your time. 
Have a nice day!
